pkg/project/operations.go

```diff
@@ -190,37 +191,50 @@ func List(ctx context.Context, opts Options) (projects []string, warnings map[st
 	return projects, warnings, nil
 }
 
-func GetDetails(ctx context.Context, opts Options, projectNames []string) (result []DetailProject, err error) {
-	entryCh := make(chan DetailProject)
-	errCh := make(chan error)
+func GetDetails(ctx context.Context, opts Options, projectNames []string) (projects []DetailProject, warnings map[string]error, err error) {
```

Comment:
Why have the warnings return when you are just coping the `err` field to the map?  Can you just make the `err` public `Err` and the client read that? I'm not sure what the extra data structure and logic buys you.

Reply:
that makes sense, removed the warnings map

Is the purpose of the warnings return in the List function that there are 2 sources they can be coming from local or hub servers and it serves to differentiate where they are coming from?